### ISSUE TITLE
global search (unified search) lost accessibility

### ISSUE THREAD

**Issue by mattlorimer** [salesagility/SuiteCRM#125]
the unified search is input is not accessible, a screen reader (for blind person) reads “Unlabeled 11 button”.
I think I tracked this to the \modules\Home\UnifiedSearchAdvancedForm.tpl

I would believe that it is line 58
<input id='searchFieldMain' class='searchField' type='text' size='80' name='query_string' value='{$query_string}'>

**Comment by bmwtourer** [comment]
no work has be performed on this?

**Comment by bmwtourer** [comment]
Thank you guys I will install over the weekend and upgrade to 7.5.2 at the same time.
I understand that the fix will not be part of currrent upgrade package and I will have to put the fix manuelly?

**Comment by mattlorimer** [comment]
It should be part of the 7.5.2 release

**Comment by bmwtourer** [comment]
Great! Thank you!

Serge Pelletier x340
819-471-4818

De : Matt Lorimer [mailto:notifications@github.com]
Envoyé : 7 mars 2016 12:27
À : salesagility/SuiteCRM <SuiteCRM@noreply.github.com>
Cc : Serge Pelletier <serge.pelletier@humanware.com>
Objet : Re: [SuiteCRM] global search (unified search) lost accessibility (#125)


Closed #125<https://github.com/salesagility/SuiteCRM/issues/125> via 4cb3bf1<https://github.com/salesagility/SuiteCRM/commit/4cb3bf1c8730fe0b5db7658d1a14e9fdb0f4bb29>.
